tests/google/adwords.py

```diff
@@ -144,17 +147,271 @@ def test__should__be_able_to_estimate_splits__when__run_with_subclass(self):
+            'isBeta': False,
+            'isZeroRowCompatible': True,
+            'enumValuePairs': [],
+            'exclusiveFields': []
+        }, {
+            'fieldName': 'ExternalCustomerId',
+            'displayFieldName': 'Customer ID',
+            'xmlAttributeName': 'customerID',
+            'fieldType': 'Long',
+            'fieldBehavior': 'ATTRIBUTE',
+            'enumValues': [],
+            'canSelect': True,
+            'canFilter': True,
+            'isEnumType': False,
+            'isBeta': False,
+            'isZeroRowCompatible': True,
+            'enumValuePairs': [],
+            'exclusiveFields': []
+        }, {
+            'fieldName': 'Id',
+            'displayFieldName': 'Keyword ID',
+            'xmlAttributeName': 'keywordID',
+            'fieldType': 'Long',
+            'fieldBehavior': 'ATTRIBUTE',
+            'enumValues': [],
+            'canSelect': True,
+            'canFilter': True,
+            'isEnumType': False,
+            'isBeta': False,
+            'isZeroRowCompatible': True,
+            'enumValuePairs': [],
+            'exclusiveFields': []
+        }, {
+            'fieldName': 'IsNegative',
+            'displayFieldName': 'Is negative',
+            'xmlAttributeName': 'isNegative',
+            'fieldType': 'Enum',
+            'fieldBehavior': 'ATTRIBUTE',
+            'enumValues': [
+                'TRUE',
+                'FALSE'
+            ],
+            'canSelect': True,
+            'canFilter': True,
+            'isEnumType': True,
+            'isBeta': False,
+            'isZeroRowCompatible': True,
+            'enumValuePairs': [
+                {
+                    'enumValue': 'TRUE',
+                    'enumDisplayValue': 'true'
+                },
+                {
+                    'enumValue': 'FALSE',
+                    'enumDisplayValue': 'false'
+                }
+            ],
+            'exclusiveFields': []
+        }, {
+            'fieldName': 'KeywordMatchType',
+            'displayFieldName': 'Match type',
+            'xmlAttributeName': 'matchType',
+            'fieldType': 'KeywordMatchType',
+            'fieldBehavior': 'ATTRIBUTE',
+            'enumValues': [
+                'EXACT',
+                'PHRASE',
+                'BROAD'
+            ],
+            'canSelect': True,
+            'canFilter': True,
+            'isEnumType': True,
+            'isBeta': False,
+            'isZeroRowCompatible': True,
+            'enumValuePairs': [
+                {
+                    'enumValue': 'EXACT',
+                    'enumDisplayValue': 'Exact'
+                },
+                {
+                    'enumValue': 'PHRASE',
+                    'enumDisplayValue': 'Phrase'
+                },
+                {
+                    'enumValue': 'BROAD',
+                    'enumDisplayValue': 'Broad'
+                }
+            ],
+            'exclusiveFields': []
+        }]
```

Comment:
It's better to move this into the resource folder rather than keeping it here as a hard coded json.